bin/make_db

```diff
@@ -44,8 +32,24 @@ def _init_db(settings):
         log.info("detected alembic_version table, skipping db initialization")
 
 
-def _init_search(settings):
+def init_search(settings):
+    # In production environments a short Elasticsearch request timeout is
+    # typically set and initializing a new search index can take longer than
+    # the timeout, so override any custom timeout with a high value.
+    os.environ["ELASTICSEARCH_CLIENT_TIMEOUT"] = "30"
+
     client = search.get_client(settings)
 
     log.info("initializing ES6 search index")
     search.init(client, check_icu_plugin=settings.get("es.check_icu_plugin", True))
+
+
+def main():
+    with bootstrap("conf/development-app.ini") as env:
+        settings = env["registry"].settings
+        init_db(settings)
+        init_search(settings)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
If you want to add a command line script to a Pyramid app this is all you need to do: add your `bin/foo.py` file (or `h/scripts/foo.py` if you want to write unit tests for it), give it a `with boostrap() as env:`, write whatever Python code you want, and call it with `tox -e dev --run-command 'python bin/foo.py'`. Each separate script is just its own standalone thing. If you want to have command line arguments use `argparse` from the standard library. The whole `bin/hypothesis` thing is totally unnecessary.